src/main/groovy/de/itemis/mps/gradle/GenerateLibrariesXml.groovy

```diff
@@ -10,6 +11,7 @@ class GenerateLibrariesXml extends DefaultTask {
     @InputFile
     File defaults
 
+    @InputFile
```

Comment:
We can't set this to `InputFile` because it must no be `null` in hat case otherwise gradle will complain hat an input is null. It definitely also needs to be `Optional`.

Reply:
Okay, will change accordingly. I was not sure about the proper annotations but gradle6 was complaining about them. :)